src/Assets/LightSkyDefense/Scripts/Enemies/EnemyAoEHeal.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+public class EnemyAoEHeal : MonoBehaviour
+{
+    public float HealAmount = 0.1f;
+    public float HealInterval = 3;
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        AoEHealWithInterval();
+    }
+
+    private void AoEHealWithInterval()
+    {
+        HealEnemies(transform.position, 0.15f, HealAmount);
```

Comment:
I think that this isn't necessary, as this will start healing when the enemy spawns. It should have full health at that point already.

Reply:
Right, but enemies around it might not.